### DISCUSSION TITLE
Multiplier bounds strategy.

### DISCUSSION THREAD

**Discussion by laat0003** [gmdsi/GMDSI_notebooks#88] [Q&A] [answered]
When using 3 scales of multipliers as demonstrated in these notebooks, should you configure them so that the only way you hit parameter ultimate bounds is if all three scales of multipliers hit their respective bounds, or does this not matter? Would this make interpretation of the parameter change summaries easier or harder?

Cheers,
Tariq

**Comment by jtwhite79** [comment, marked as answer]
I think that is the general idea - you want to split up the property uncertainty so that each scale gets an appropriate variance and then the ultimate bounds are just a sanity check right before the model runs.  I tend to use the array parameter summary stats ([https://github.com/pypest/pyemu/blob/9b2b53bdcfbc3c94487b17b9cec3e63b0765de18/pyemu/utils/helpers.py#L1855](https://github.com/pypest/pyemu/blob/9b2b53bdcfbc3c94487b17b9cec3e63b0765de18/pyemu/utils/helpers.py#L1855)) as observations to keep an eye in the actual array values that the model is seeing.

This does bring up something Ive run into in practice tho:  If the ultimate bounds dont jive completely with the multiplier values times original input values (meaning the ultimate bound enforcement is resetting a lot of values), this can make the inversion more nonlinear (and therefore harder to solve).  So those array summary stats become important for watching for this condition...would probably be good to include those stats in the notebooks at some point...

**Reply by laat0003** [reply]
Thanks @jtwhite79 for pointing that out. I didn't use those summary stats in my previous project as observations and may have had some of the issues with non-linearity that you mention. Instead, I was trying to decipher the model input arrays from multiplier behavior reported in the plot_utils.ensemble_change_summary() post IES - which was really hurting my head.  

I am in the process of setting up another model at the moment for a different project so will use the stats as observations this time.  
Cheers,
